### ISSUE TITLE
dragging content struggles when heavy markup

### ISSUE THREAD

**Issue by ax2000** [jakiestfu/Snap.js#84]
When the content div has lots of markup it gets very difficult to swipe to open. It's fine by clicking on the button to open the panel, but gets really heavy to open or close it by dragging. Tried to overflow hidden the content but doesn't seem to make much difference. 
Any thoughts I could make it less heavy to drag?

**Comment by jakiestfu** [comment]
How many nodes are we talking here? Are we talking. About ~ 100 child nodes? 1,000? Each with links, maybe an image, text, etc?

**Comment by ax2000** [comment]
Yes it starts with about 30 + list elements width percentage width (don't know if would be recalculating widths every time),  each element with an  image, different links, icons etc.. It's definitely not the ideal content for the browser to drag. Other pages with common content work flawlessly.

**Comment by jakiestfu** [comment]
Definitely interested in looking into this. Thanks for the heads up (and
kind words!)

**Comment by ax2000** [comment]
Nice!
Thank you for your great work!
